main/templates/main/hospital_filter.html

```diff
@@ -3,75 +3,76 @@
+  </a>
+  <div class="flex text-center font-bold bg-white py-3 rounded-b-lg h-16">
+    <div class="w-1/2 px-2 border-l-8 border-hospitalaid-blue-light">
+      <span id="id_supply" class="flex justify-center text-3xl fg-hospitalaid-black"></span>
+      <span class="flex justify-center text-sm text-gray-500 tracking-widest leading-normal">SUPPLY</span>
     </div>
-    <div class="flex text-center font-bold py-2 bg-white">
-      <div class="w-1/2 border-l-8 border-red-300 p-2 px-4">
-        <div class="text-3xl" id="repair">
-        </div>
-        <div class="text-lg text-gray-500">
-          REPAIR
-        </div>
-      </div>
-      <div class="w-1/2 border-r-8 border-blue-300 p-2 px-4">
-        <div class="text-3xl" id="supply">
-        </div>
-        <div class="text-lg text-gray-500">
-          SUPPLY
-        </div>
-      </div>
+    <div class="border border-hospitalaid-gray-light">
+    </div>
+    <div class="w-1/2 px-2 border-r-8 border-hospitalaid-red">
+      <span id="id_repair" class="flex justify-center text-3xl fg-hospitalaid-black"></span>
+      <span class="flex justify-center text-sm text-gray-500 tracking-widest leading-normal">REPAIR</span>
     </div>
   </div>
 </template>
-<div id="map"></div>
+
+<div id="id_map"></div>
 {% endblock content %}
 
 {% block js %}
-    <script src="{% static "/js/leaflet.js" %}"></script>
-    <script charset="utf-8">
-      var map_data = [
-        {% for h in object_list %}
-        {
-          name: "{{ h.name }}",
-          town: "{{ h.city }}",
-          country: "{{ h.country }}",
-          latitude: {{ h.latitude|default:0.0|unlocalize }},
-          longitude: {{ h.longitude|default:0.0|unlocalize }},
-          text: "{% for aid in h.aidrequest_set.all %}<p>{{ aid }}</p>{% endfor %}",
-          link: "{% url 'hospitalfordonor_detail' h.id %}",
-        },
-        {% endfor %}
-      ]
-    </script>
-    <script src="{% static "/js/main.js" %}"></script>
+<!-- Make sure you put this AFTER Leaflet's CSS -->
+<script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
+        integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
+        crossorigin=""></script>
+<script src="https://cdn.jsdelivr.net/npm/leaflet-easybutton@2/src/easy-button.js"></script>
+<script charset="utf-8">
+  var map_data = [
+      {% for hospital in object_list %}
+      {
+          name: "{{ hospital.name }}",
+          town: "{{ hospital.city }}",
+          country: "{{ hospital.country }}",
+          latitude: {{ hospital.latitude|default:0.0|unlocalize }},
+          longitude: {{ hospital.longitude|default:0.0|unlocalize }},
+          text: "{% for aid in hospital.aidrequest_set.all %}{{ aid }}{% endfor %}",
+          link: "{% url 'hospitalfordonor_detail' hospital.id %}",
+          supply_count: "{{ hospital.aidrequest_supply_count }}",
+          repair_count: "{{ hospital.aidrequest_repair_count }}",
```

Comment:
We should get rid of the fields that we do not use in the popup. Text, town and country.

Reply:
Yes. This is a work in progress. Next release there should be other improvements/cleanup.